api/v1alpha1/nodepool_types.go

```diff
@@ -134,6 +134,17 @@ type NodePoolSpec struct {
 	// https://github.com/kubernetes-sigs/cluster-api/issues/5880
 	// +optional
 	NodeDrainTimeout *metav1.Duration `json:"nodeDrainTimeout,omitempty"`
+
+	// TunedConfig is a list of references to ConfigMaps containing serialized
+	// Tuned resources to define the tuning configuration to be applied to
+	// nodes in the NodePool. The Tuned API is defined here:
+	//
+	// https://github.com/openshift/cluster-node-tuning-operator/blob/2c76314fb3cc8f12aef4a0dcd67ddc3677d5b54f/pkg/apis/tuned/v1/tuned_types.go
+	//
+	// Each ConfigMap must have a single key named "tuned" whose value is the
+	// JSON or YAML of a serialized Tuned.
+	// +kubebuilder:validation:Optional
+	TunedConfig []corev1.LocalObjectReference `json:"tunedConfig,omitempty"`
```

Comment:
this field and `Config` are examples of Linux only fields.  Do we want to start adding a Linux specific struct to encapsulate these fields?  It would potentially make Windows easier in future.

Reply:
That seems reasonable to me... @csrwng what do you think? 

Is this something we want to address in this PR, or before this PR merges?

Reply:
Let's address in a separate PR. @derekwaynecarr I can introduce the OS-specific struct in the v1beta1 API that I'm working on right now. We need to have a way to migrate the existing API given that it's already in use as is.